cis194/week3/daniel-deng/src/Exercise03.hs

```diff
@@ -0,0 +1,24 @@
+module CIS194.Homework03.Exercise03 (histogram) where
+
+import Data.List
+
+stars :: Int -> String
+stars = flip replicate '*'
+
+spaces :: Int -> String
+spaces = flip replicate ' '
+
+starLine :: Int -> String
+starLine nTimes = spaces (10 - nTimes) ++ stars nTimes
+
+count :: Eq a => a -> [a] -> Int
+count x = length . filter (==x)
+
+starLines :: [Integer] -> [String]
+starLines xs = filter (/= spaces 10)
+             . transpose
+             . fmap (starLine . flip count xs)
```

Comment:
@shineli1984 Which one below is more used in production code? Or is there a third way in day-to-day code?

```suggestion
             . fmap (starLine . (`count` xs))
```